fili-core/src/main/java/com/yahoo/bard/webservice/web/SlicesApiRequest.java

```diff
@@ -165,7 +164,7 @@ protected SlicesApiRequest() {
             throw new BadApiRequestException(msg);
         }
 
-        Map<Column, List<Interval>> columnCache = table.getAvailability().getAvailableIntervals();
+        Map<Column, Set<Interval>> columnCache = table.getAllAvailableIntervals();
```

Comment:
I am trying to push towards the conclusion that we do have a choice. [this](https://github.com/yahoo/fili/blob/master/fili-core/src/test/groovy/com/yahoo/bard/webservice/web/endpoints/BaseDataServletComponentSpec.groovy#L120) thing exists for this kind of reason right? If ordering matters for a component, the test should be written accordingly to enforce ordering. If ordering does not matter, the tests should be written in a way that order does not matter. The ordering is a part of the test right?